### ISSUE TITLE
Single data on chart makes application crash 

### ISSUE THREAD

**Issue by N7ghtm4r3** [ehsannarmani/ComposeCharts#46]
Hi, I'm testing my application with the library, I have the possibility to have a single value on the chart, I know that it can look like a strange scenario, but when I drag to display each point on the chart the applications crashes with the following exception: 

```java
java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
 at jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
 at jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
 at jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
 at java.util.Objects.checkIndex(Objects.java:359)
 at java.util.ArrayList.get(ArrayList.java:434)
 at ir.ehsannarmani.compose_charts.extensions.line_chart.PopupHelperKt.getPopupValue-wzdHmys(PopupHelper.kt:50)
 at ir.ehsannarmani.compose_charts.LineChartKt$LineChart$13$2$1$1.invokeSuspend$lambda$3(LineChart.kt:267)
 at ir.ehsannarmani.compose_charts.LineChartKt$LineChart$13$2$1$1.$r8$lambda$UPcUeQstgzkemKjhxUjQhCK3atc(Unknown Source:0)
 at ir.ehsannarmani.compose_charts.LineChartKt$LineChart$13$2$1$1$$ExternalSyntheticLambda1.invoke(Unknown Source:32)
 at androidx.compose.foundation.gestures.DragGestureDetectorKt$detectDragGestures$9.invokeSuspend(DragGestureDetector.kt:274)
 at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
 at kotlinx.coroutines.DispatchedTaskKt.resume(DispatchedTask.kt:165)
 at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:154)
 at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
 at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
 at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
 at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
```

is the normal behavior have to happen with a single point on the chart?

The chart configuration is the following:

```kotlin

private val axisProperties = GridProperties.AxisProperties(
    enabled = false
)

....

LineChart(
  modifier = Modifier
        .fillMaxSize(),
    gridProperties = GridProperties(
        enabled = false,
        xAxisProperties = axisProperties,
        yAxisProperties = axisProperties
    ),
    dividerProperties = DividerProperties(
        enabled = false,
        yAxisProperties = LineProperties(
            enabled = false
        )
    ),
    indicatorProperties = HorizontalIndicatorProperties(
        enabled = false
    ),
    labelHelperProperties = LabelHelperProperties(
        enabled = false
    ),
    data = chartData // for example listOf(2.0)
 )
```

**Comment by ehsannarmani** [comment]
Hi @N7ghtm4r3 
Which version are you using?

**Comment by N7ghtm4r3** [comment]
Sorry, I forgot to write it.
I'm using the `0.0.14` version

**Comment by ehsannarmani** [comment]
I think its solved in 0.0.16
Please upgrade and let me know the problem is solved or still crashing.
